lib/messages/message.js

```diff
@@ -12,7 +12,7 @@ var Hash = bitcore.crypto.Hash;
  * @param {Number=} options.magicNumber
  * @constructor
  */
-function Message(options) {
+function Message(arg, options) {
   this.command = options.command;
   this.magicNumber = options.magicNumber;
 }
```

Comment:
So that it matches the same order of arguments of the inherited messages, but it may not be necessary.